datalad/utils.py

```diff
@@ -800,6 +800,7 @@ def knows_annex(path):
     """
     from os.path import exists
     if not exists(path):
+        lgr.debug("No annex: test path doesn't exist")
```

Comment:
Please usually include the actual value in the debug msg, i.e. actual `path` here